ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+## Description
+
+Provide a description of your issue.
+For any suggestion, please address them on the [forum](https://forum.freegamedev.net/viewforum.php?f=16).
+For translations, go to [transifex](https://www.transifex.com/supertuxkart/supertuxkart/).
+
+## Steps to reproduce
+
+## Configuration
+
+STK release version:
+
+STK Source (ppa, distribution package, official bin,...):
+
+System: 
+
+Graphics card: 
+
+CPU: 
+
+Gamepads/keyboards models if related to the issue:
+
+## Additional informations
+
+Please provide stdout.log, it is located in
+
+* %appdata%\supertuxkart\0.8.2 (windows)
+* ~/.config/supertuxkart/0.8.2 (linux, mac and other unix based systems)
+
+stdout.log
+```
+
+```
+
+If your issue is related to the input config (gamepads, keyboards), please provide your file input.xml located in the same directory as stdout.log.
+
+input.xml
+```xml
+
+```
+
+
+If a crash occured, please provide a backtrace
```

Comment:
Most users will not understand what this is. I'd say, "if an error message was shown, please copy paste the complete error message or a screenshot of it"